migrate/v0_17/migrate.go

```diff
@@ -0,0 +1,50 @@
+package v0_17
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	genutiltypes "github.com/cosmos/cosmos-sdk/x/genutil/types"
+	"github.com/kava-labs/kava/app"
+	tmtypes "github.com/tendermint/tendermint/types"
+)
+
+var (
+	GenesisTime = time.Date(2022, 5, 10, 16, 0, 0, 0, time.UTC)
```

Comment:
```suggestion
	GenesisTime = time.Date(2022, 5, 10, 17, 0, 0, 0, time.UTC)
```

17 UTC is confirmed genesis time for now. Let's add a TODO to verify before release in case anything changes.